themes/yt/template.html

```diff
@@ -90,6 +90,41 @@
     background-repeat: repeat-x;
     background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAACqCAYAAABsziWkAAAAAXNSR0IArs4c6QAAAQVJREFUOE9lyNdHBQAAhfHb3nvvuu2997jNe29TJJEkkkgSSSSJJJJEEkkiifRH5jsP56Xz8PM5gcC/xfDEmjhKxEOCSaREEiSbFEqkQppJpzJMJiWyINvkUCIX8kw+JQqg0BRRxaaEEqVQZsopUQGVpooS1VBjglStqaNEPTSYRko0QbNpoUQrtJl2qsN0UqILuk0PJXqhz/RTYgAGzRA1bEYoMQpjZpwSExAyk5SYgmkzQ82aOUqEIWKilJiHBbNIiSVYhhVYhTVYhw3YhC3Yhh3YhT3YhwM4hCM4hhM4hTM4hwu4hCu4hhu4hTu4hwd4hCd4hhd4hTd4hw/4hC/4hh/4/QM2/id28uIEJAAAAABJRU5ErkJggg==");
   }
+
+  media-captions-button {
+    width: 44px;
+    height: 44px;
+    padding: 0;
+  }
+
+  /* Hide captions button when no subtitles are available */
+  media-captions-button:not([mediasubtitleslist]) svg {
+    opacity: 0.3;
+  }
+
+  media-captions-button svg {
+    width: 44px;
+    height: 44px;
+    padding: 0;
+  }
+
+  media-captions-button[mediasubtitleslist]:after {
+    content: "";
+    display: block;
+    position: absolute;
+    width: 0;
+    height: 3px;
+    border-radius: 3px;
+    background-color: #f00;
+    bottom: 6px;
+
+    transition: left 0.25s cubic-bezier(0, 0, 0.2, 1),
+      width 0.25s cubic-bezier(0, 0, 0.2, 1);
```

Comment:
```suggestion
    transition: left .1s cubic-bezier(0.4, 0, 1, 1), 
      width .1s cubic-bezier(0.4, 0, 1, 1);
```

the hiding is faster on YT